Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/Instance.h

```diff
@@ -177,6 +178,13 @@ namespace AzToolsFramework
             AZStd::pair<Instance*, AZ::EntityId> GetInstanceAndEntityIdFromAliasPath(AliasPathView relativeAliasPath);
             AZStd::pair<const Instance*, AZ::EntityId> GetInstanceAndEntityIdFromAliasPath(AliasPathView relativeAliasPath) const;
 
+            /**
+             * Gets the nested instance with the given alias, if present.
+             *
+             * @param alias The alias of the instance we want to get.
+             * @return The reference of the instance with the alias queried, or a null reference if there's no match.
+             */
+            InstanceOptionalReference GetNestedInstanceByAlias(InstanceAlias alias) const;
```

Comment:
There is already a FindNestedInstance() that you can use instead of this

Reply:
Not sure how I missed it :'D Fixed, thanks!